docs/_static/auth/iam.yaml

```diff
@@ -562,6 +562,290 @@ paths:
+                  subject_syntax_types_supported:
+                    type: array
+                    items:
+                      type: string
+                    description: "A JSON array of strings representing URI scheme identifiers and optionally method names of supported Subject Syntax Types defined in Section 8. When Subject Syntax Type is JWK Thumbprint, a valid value is urn:ietf:params:oauth:jwk-thumbprint defined in [RFC9278]. When Subject Syntax Type is Decentralized Identifier, valid values MUST be a did: prefix followed by a supported DID method without a : suffix. For example, support for the DID method with a method-name \"example\" would be represented by did:example. Support for all DID methods listed in Section 13 of [DID_Specification_Registries] is indicated by sending did without any method-name."
+                  id_token_types_supported:
+                    type: array
+                    items:
+                      type: string
+                    description: | 
+                      A JSON array of strings containing the list of ID Token types supported by the OP, the default value is attester_signed_id_token. The ID Token types defined in this specification are:
+                      subject_signed_id_token: Self-Issued ID Token, i.e. the ID Token is signed with key material under the End-User's control.
+                      attester_signed_id_token: the ID Token is issued by the party operating the OP, i.e. this is the classical ID Token as defined in [OpenID.Core].
+        default:
+          $ref: '../common/error_response.yaml'
+  /iam/oid4vci/dcr/{sessionId}/authorize:
+    get:
+      operationId: dcrOid4vpAuthorize
+      summary: Authorize in the Dynamic Credential Request OID4VP flow.
+      tags:
+        - dcr
+        - oid4vp
+      parameters:
+        - name: sessionId
+          in: path
+          required: true
+          description: The sessionId of the callback request.
+          schema:
+            type: string
+            example: 55d7a35d-d7bf-436f-80f7-3fef4077f8a8
+        - name: request
+          in: query
+          required: true
+          description: | 
+            Should be a signed request object as JWT with the following required fields:
+              - response_type : vp_token
+              - client_id : did value
+              - client_id_scheme : "did"
+              - redirect_uri: the uri to redirect to
+              - state: state 
+              - nonce: unique 
+              - presentation_definition or presentation_definition_uri
+              - user_hint
+          schema:
+            type: string
+      responses:
+        '302':
+          description: |
+            The user-agent is redirected to the redirectURL submitted at the start-oid4vci-issuance request.
+          headers:
+            Location:
+              schema:
+                type: string
+                format: uri
+  /iam/oid4vci/{sessionId}/callback:
```

Comment:
must be callback without sessionId in path (not allowed by oauth2).
To transfer state, use the `state` query param.

Reply:
But with `id` param: `/iam/{id}/oid4vci/callback`

You could also use the general callback url but then would have to figure out what flow was started based on state information.

Reply:
I think state will do, as it is required is set on the auth request.

>    state
         REQUIRED if the "state" parameter was present in the client
         authorization request.  The exact value received from the
         client.